### PR TITLE
Split by files fix

### PR DESCRIPTION
**Pull request by mesmith75**
For issue #727 
Just changed the isType to isinstance for GangaList type datasets.

### CODE REVIEW

File: python/GangaLHCb/Lib/Splitters/SplitByFiles.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def _create_subjob(self, job, dataset):
                     logger.error("Unkown file-type %s, cannot perform split with file %s" % (type(i), str(i)))
                     from Ganga.Core.exceptions import GangaException
                     raise GangaException("Unkown file-type %s, cannot perform split with file %s" % (type(i), str(i)))
-        elif type(dataset) == type([]) or isType(dataset, GangaList()):
+        elif type(dataset) == type([]) or isinstance(dataset, GangaList):
```
Comment (alexanderrichards) [inline python/GangaLHCb/Lib/Splitters/SplitByFiles.py]: this is better written in a combined `isinstance` as follows:

```python
elif isinstance(dataset, (list, GangaList))
```

**Comment by mesmith75** [comment]
@alexanderrichards good point, done.

**Comment by rob-c** [comment]
Might as well merge this in before 6.1.25 I think

**Comment by mesmith75** [comment]
@rob-c I agree

**Comment by drmarkwslater** [comment]
👍

**Pull request merged**